dev-java/jcommander/jcommander-1.82.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source test"
+JAVA_TESTING_FRAMEWORKS="testng"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Command line parsing framework for Java"
+HOMEPAGE="https://github.com/cbeust/jcommander"
+SRC_URI="https://github.com/cbeust/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+
+DEPEND=">=virtual/jdk-1.8:*
+	test? ( dev-java/jackson-annotations:2 )
+"
+RDEPEND=">=virtual/jre-1.8:*"
+
+S="${WORKDIR}/${P}"
```

Comment:
That's the default value for `S`, isn't it? If so, then this can be omitted.

Reply:
java-pkg-simple.eclass defaults to `S="${WORKDIR}"`.